### PR TITLE
Move the token to an encrypted data bag and allow the conf to be an incl...

### PR DESCRIPTION
**Pull request by cjs226**
...ude

### CODE REVIEW

**Comment by mveitas** [comment]
A few quick things:

1) Do you mind breaking this into 2 pull requests, one for the encrypted databag and another for the change to the the template file?
2) Can you please add or modify any unit tests as necessary? I need to add in a task for Travis to execute the unit tests.

**Comment by mveitas** [comment]
@cjs226 Do you plan on submitting another PR? If not, I will add in the support for the encrypted databags

**Comment by cjs226** [comment]
Yes, I plan on doing so.  Give me a day or so to figure out unit tests, etc.

**Comment by mveitas** [comment]
I am reworking some of our cookbooks that we use at work (loggly-rsyslog is one of them) and I can easily make the change to use an encrypted databag if you'd like.

File: attributes/default.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
-default['loggly']['token'] = nil
-default['loggly']['tags'] = []
+loggly_token = Chef::EncryptedDataBagItem.load("loggly", "token")
+default['loggly']['token'] = loggly_token['token']
```
Comment (mveitas) [inline attributes/default.rb:2]: Pulling the data from the encrypted data bag and then storing it in as an attribute on a node exposes the data to anybody that can view the node details. The better approach is to store this and then pass it in as a variable to the template.

**Pull request closed without merging**